src/util.cpp

```diff
@@ -938,6 +946,22 @@ bool hasNonPublicParam(const CXXMethodDecl *decl)
     return false;
 }
 
+bool getPythonOperatorSpelling(OverloadedOperatorKind kind, std::string &str)
+{
+    switch (kind)
+    {
+        case OverloadedOperatorKind::OO_Plus:
+            str = "__add__";
+            return true;
+        case OverloadedOperatorKind::OO_Star:
+            str = "__mul__";
+            return true;
```

Comment:
I wonder if there is any way we can resolve this to a lookup in the binding configuration.  I think that would be best so that different bindings could map these in different ways.  But I am not sure what the best approach is.

Reply:
I basically like this idea as it would make chimera more flexible for broader language binding support as you pointed out. Just don't have a good idea how to implement. Especially adding a map to the template wouldn't be straightforward since mustache is a logic-less template.

Maybe using a new configuration YAML block or file for language specifics?